src/test/com/abstract_calss/Hawk.java

```diff
@@ -0,0 +1,12 @@
+package test.com.abstract_calss;
+
+public class Hawk extends Animal {
+    @Override
+    void move() {
+        System.out.println("fly");
+    }
+    @Override
+    void talk() {
+
```

Comment:
Let's assume that it's making sound and not talking, from google: 
Adults make a hoarse, screaming kee-eeeee-arr. It lasts 2-3 seconds and is usually given while soaring. During courtship, they also make a shrill chwirk, sometimes giving several of these calls in a row. 
Try to enjoy the process, you're learning not by going through tasks fast but by doing them well and by training correct behavior